lib/widget/custom_button_widget.dart

```diff
@@ -15,13 +15,14 @@ class CustomButtonWidget extends StatelessWidget {
   Widget build(BuildContext context) {
     return SizedBox(
       width: double.infinity,
-      height: Dimens.space56,
+      height: Dimens.customButtonHeight,
```

Comment:
What do you think about the new name of this custom button to make it more sense, such as `PrimaryButtonWidget` or `RoundedButtonWidget`?
Also `customButtonHeight` and `customButtonBorderRadius`